test cases/common/262 configure file output format/compare.py

```diff
@@ -0,0 +1,5 @@
+from pathlib import Path
+import sys
+
+are_equals = Path(sys.argv[1]).read_text(encoding='utf-8') == Path(sys.argv[2]).read_text(encoding='utf-8')
+sys.exit(0 if are_equals else 1)
```

Comment:
```python
with open(sys.argv[1], 'rb') as f, open(sys.argv[2], 'rb') as g:
    if f.read() != g.read():
        sys.exit('contents are not equal')
```

- there's no real reason to open this as unicode text, since what we are checking is that meson produces the same file, but we don't actually care about interpreting what's in it
- the sys.exit seems complicated and hard to read compared to exiting with a message in an if block
- it is not clear to me what benefit pathlib brings here as no path manipulation is used and even the method names are unnecessarily longer IMO

Reply:
IIRC, I did this in text mode instead of using filecmp, because of line endings that are different on Windows. By opening the file in text mode, Python uses universal newlines, and the files compare equally even if the reference file is from a different platform.

Reply:
Huh. I did not think of that.

I think this just matters for C/nasm, right? I wonder if it makes sense to guarantee consistency when generating those files by opening them with `newline=''`

Reply:
I think it is easier / simpler to keep the test in text mode for now. After all, this is just a script for a test. I rewrote it like you proposed.